test/index.js

```diff
@@ -1197,7 +1197,7 @@ describe('flash()', () => {
 
                     server.inject({ method: 'GET', url: '/2', headers: { cookie: cookie[1] } }, (res2) => {
 
-                        expect(res2.result.session._flash.error).to.not.exist();
+                        expect(res2.result._flash).to.not.exist();
```

Comment:
I don't think this required any change.  `res2.result._flash` is guaranteed not to exist based upon how the handler replies.  `_flash` lives on `request.yar._store`.  Other tests that reply with `reply(request.yar._store)` do correctly check `res.result._flash`, which may be the reason this change was made.